srtp/srtp.c

```diff
@@ -3202,6 +3206,9 @@ static void srtp_calc_aead_iv_srtcp(srtp_session_keys_t *session_keys, v128_t *i
     in.v16[0] = 0;
     memcpy(&in.v16[1], &hdr->ssrc, 4); /* still in network order! */
     in.v16[3] = 0;
+    if (seq_num & 0x80000000) { /* the most significant bit should be zero */
```

Comment:
Tacking on 'UL' I think would be a good thing. In that case we get compilation warnings if we try to assign it to a 'int32_t' or something else that might overflow. 

Would not produce a warning for `&` though, but it is an almost free low effort piece of defensive programming - normally a good thing!